eventMacro/Core.pm

```diff
@@ -70,24 +68,22 @@ sub clean_hooks {
 	foreach (@{$self->{Hook_Handles}}) {Plugins::delHook($_)}
 }
 
-sub is_paused {
-	my ($self) = @_;
-	return ( $self->{Paused} ? 1 : 0 );
-}
-
-sub is_executing {
-	my ($self) = @_;
-	return ( $self->{Status} ? 1 : 0 );
-}
-
-sub pause {
-	my ($self) = @_;
-	$self->{Paused} = PAUSED;
+sub set_automacro_checking_status {
+	my ($self, $status) = @_;
+	
+	if ($self->{Automacros_Checking_Status} == CHECKING_AUTOMACROS) {
```

Comment:
There was a return for this case in the previous commit, I removed it by accident.